src/Api/Request/InitializeService.php

```diff
@@ -50,11 +53,8 @@ public function initialize(InitializeRequest $initializeRequest, $isBusinessLice
         }
         $response = $this->apiRequest->post(
             $initializeApi,
-            [
-                'body' => json_encode($initializeRequest->getAsArray()),
-            ]
+            json_encode($initializeRequest->getAsArray()),
```

Comment:
is 'body' not required key?

Reply:
maybe for guzzle we needed to add body key that has json, but for unirest it just accepts the json itself.
TLDR: didn't work with body key, works without it